Xamarin.Forms.Core/Markup/GridRowColEnums.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.Globalization;
+
+namespace Xamarin.Forms.Markup
+{
+	public static class GridRowColEnums
+	{
+		public static GridLength Auto => GridLength.Auto;
+
+		public static GridLength Star => GridLength.Star;
+
+		public static class Columns
+		{
+			public static ColumnDefinitionCollection Define<TEnum>(params (TEnum name, GridLength width)[] cols) where TEnum : Enum
```

Comment:
We could add that overload for devs who want to use numbers for their rows and columns instead of enums, but still want a concise way to define rows and columns.

I have not had or encountered this need, just love enums, but it think it won't hurt to add it.
What do you say, add it?

Reply:
Added. 
Also renamed `GridRowColEnums` to `GridRowsColumns` and documented in PR description